### PR TITLE
Project set-up

### PR DESCRIPTION
**Pull request by danmalmx**
##PT-chore: https://www.pivotaltracker.com/story/show/161218347

* Initialised rails with postgresql
* Skipped test
* Added gems and bundled

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.2
+
+    # Settings in config/environments/* take precedence over those specified here.
+    # Application configuration can go into files in config/initializers
+    # -- all .rb files in that directory are automatically loaded after loading
+    # the framework and any gems in your application.
+
+    # Don't generate system test files.
```
Comment (Rinbo) [inline config/application.rb]: Remove comments

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+# On Heroku and other platform providers, you may have a full connection URL
+# available as an environment variable. For example:
+#
+#   DATABASE_URL="postgres://myuser:mypass@localhost/somedatabase"
+#
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
```
Comment (Rinbo) [inline config/database.yml]: Remove comments

File: config/environments/development.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  # Suppress logger output for asset requests.
+  config.assets.quiet = true
+
+  # Raises error for missing translations
+  # config.action_view.raise_on_missing_translations = true
+
+  # Use an evented file watcher to asynchronously detect changes in source code,
+  # routes, locales, etc. This feature depends on the listen gem.
+  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+end
```
Comment (Rinbo) [inline config/environments/development.rb:27]: Remove comments

File: config/environments/production.rb [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+  # Send deprecation notices to registered listeners.
+  config.active_support.deprecation = :notify
+
+  # Use default logging formatter so that PID and timestamp are not suppressed.
+  config.log_formatter = ::Logger::Formatter.new
+
+  # Use a different logger for distributed setups.
+  # require 'syslog/logger'
+  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
```
Comment (Rinbo) [inline config/environments/production.rb]: Remove comments

File: Gemfile [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  gem 'pry-byebug'
+  gem 'pry-rails'
+  gem 'rspec-rails'
+  gem 'shoulda-matchers'
+  gem 'factory_bot_rails'
+  gem 'cucumber-rails', '~> 1.6', require: false
+  gem 'database_cleaner'
+  gem 'chromedriver-helper'
+  gem 'selenium-webdriver'
+  gem 'faker'
```
Comment (Rinbo) [inline Gemfile]: Remove Faker

File: Gemfile [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  gem 'devise'
```
Comment (Rinbo) [inline Gemfile]: remove devise

File: Gemfile [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  gem 'coveralls', require: false
+  gem 'pry-byebug'
+  gem 'pry-rails'
+  gem 'rspec-rails'
+  gem 'shoulda-matchers'
+  gem 'factory_bot_rails'
+  gem 'cucumber-rails', '~> 1.6', require: false
+  gem 'database_cleaner'
+  gem 'chromedriver-helper'
+  gem 'selenium-webdriver'
```
Comment (Rinbo) [inline Gemfile]: Remove testing gems line 20-29

File: Gemfile [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+gem 'sprockets', '~> 3.7.2'
+gem 'ffi', '~> 1.9.24'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'turbolinks', '~> 5'
+gem 'bootsnap', '>= 1.1.0', require: false
+gem 'webpacker', '~> 3.5'
+gem 'tailwindcss', '~> 0.2.0'
```
Comment (Rinbo) [inline Gemfile]: Remove tailwind

**Review by magnus-thor — APPROVED** [review]

**Pull request merged**